WasatchNET/EEPROM.cs

```diff
@@ -1042,7 +1075,12 @@ public bool write()
                 if (!ParseData.writeBool(hasCooling, pages[0], 36)) return false;
                 if (!ParseData.writeBool(hasBattery, pages[0], 37)) return false;
                 if (!ParseData.writeBool(hasLaser, pages[0], 38)) return false;
-                if (!ParseData.writeUInt16(featureMask.toUInt16(), pages[0], 39)) return false;
+
+                if (format >= 9)
+                    if (!ParseData.writeUInt16(featureMask.toUInt16(), pages[0], 39)) return false;
+                else
+                    if (!ParseData.writeUInt16(excitationNM, pages[0], 39)) return false;

```

Comment:
**This is root cause for CAR0153.**  Missed in initial testing and code review :-(